.docker/Dockerfile

```diff
@@ -0,0 +1,58 @@
+ARG POSTGIS_VERSION=9.6-2.5

```

Comment:
Why not use https://github.com/kartoza/docker-postgis/tree/11.0-2.5 instead? That should ship a newer python version too.

Reply:
I was used to mdillon, but saw it was moved to the postgis organization, so it's what's closest to an official release ?
But actually, I'm starting to think merging it in 1 container was not the best idea... basically it just saves 1 command line, and makes it much harder to swap the postgis image... Maybe we can change that at some point, so that we could use whatever postgis image/tag we want.

Reply:
But yes indeed it's based on buster, so python wouldn't have been an issue...